tests/schedulers/test_schedulers.py

```diff
@@ -481,8 +481,8 @@ def test_step_shape(self):
 
         num_inference_steps = kwargs.pop("num_inference_steps", None)
 
-        timestep_0 = 0
-        timestep_1 = 1
+        timestep_0 = 1
```

Comment:
with `nonzero()` search, we can feed the step function with arbitrary timesteps each time and it would work

now we've switched to a `_step_index` counter that we increase at each step, we have to feed the step consecutive timesteps